great_expectations/agent/agent.py

```diff
@@ -0,0 +1,122 @@
+import os
+from typing import Dict, Optional
+
+from pydantic.dataclasses import dataclass
+from pydantic import AmqpDsn
+from great_expectations import get_context
+from great_expectations.agent.event_handler import ShutdownRequest, EventHandler
+from great_expectations.agent.message_service.rabbit_mq_client import (
+    RabbitMQClient,
+    ClientError,
+)
+from great_expectations.agent.message_service.subscriber import (
+    Subscriber,
+    OnMessageCallback,
+    SubscriberError,
+)
+from great_expectations.agent.models import Event
+from great_expectations.data_context import CloudDataContext
+
+
+HandlerMap = Dict[str, OnMessageCallback]
```

Comment:
```suggestion
HandlerMap: TypeAlias = Dict[str, OnMessageCallback]
```

Reply:
added in [87fa5a5](https://github.com/great-expectations/great_expectations/pull/7837/commits/87fa5a54b9427683e0fde954c3a5051c1516b511)

Reply:
Did you revert this?

You probably need to import this from `typing_extensions`

Reply:
this was causing an error on python 3.8 so i reverted it